terms/structuralriskminimization_maf.tex

```diff
@@ -0,0 +1,99 @@
+\chapter{\textit{}Structural Risk Minimization}
+\label{ch:structuralriskminimization}\index{structuralriskminimization}
+
+
+\section Overview \newline \index{overview}
+
+Structural Risk Minimization (SRM) is a technique first developed by Vapnik \& Chervonenkis in their book from 1974  \cite{vapnikchervonenkis1}. This technique is used to identify the function $f(x)$ that solves our machine learning problem $y=f(x)$. SRM attempts to find a balance between training our model accurately, and not having an overly complex solution.
```

Comment:
DO NOT HARDCODE AUTHORS use citep or citet intead (Vapnik \& Chervonenkis).  The first sentence has zero information content.